Framework/PythonInterface/plugins/algorithms/CompareSampleLogs.py

```diff
@@ -39,8 +41,8 @@ def PyInit(self):
 
         self.declareProperty(StringArrayProperty(name="SampleLogs", direction=Direction.Input, validator=validator),
                              doc="Comma separated list of sample logs to compare.")
-        self.declareProperty("Tolerance", 1e-3, validator=FloatBoundedValidator(lower=1e-7, upper=1.0),
-                             doc="Tolerance for comparison of double values.")
+        self.declareProperty("Tolerance", 1e-3, validator=FloatBoundedValidator(lower=1e-7, upper=100.0),
```

Comment:
Load the CNCS_7860 from doc tests. Run `FilterBadPulses` to eliminate al the  values when the accelerator was down. The `proton_charge` log is in picoCoulombs, with an average of 1.81E7, and a standard deviation of about 82000. It all depends on the units. I would just leave a lower bound